### ISSUE TITLE
Login Screen Background Image goes beyond footer

### ISSUE THREAD

**Issue by bslima** [bmbrands/moodle-theme_elegance#74]
I'm using Moodle 2.7.1. on Chrome for Mac 31.

I have a retina display with 1680x1050 resolution and at the login screen the footer is not in the bottom.

The background image goes beyond the footer.

Here it is a image sample.

![untitled](https://cloud.githubusercontent.com/assets/399274/4140471/6c25c1d4-33a5-11e4-86ba-ffbc30a2062e.png)

**Comment by vistilarsen** [comment]
Excact same problem with the same build! It only appears on large screens.

**Comment by richardschwarz** [comment]
Yep. Need to alter the layout file to push the footer to the bottom of the page or content, whichever is lower. Looking at doing that in a fork. Will offer a pull request when I can get it to reliably work.
